lavalink/__init__.py

```diff
@@ -49,6 +50,12 @@ def add_event_hook(hook, event: Event = None):
         The event the hook belongs to. This will dispatch when that specific event is
         dispatched.
     """
+    if event is not None or not isinstance(event, Event):
+        raise TypeError('Event parameter is not of type Event or None')
+
+    if not callable(hook) or inspect.iscoroutinefunction(hook):
+        raise TypeError('Hook is not callable or a coroutine')
```

Comment:
Missing `not` after `or`? I thought this change meant non-coroutine hooks couldn't be used?

Reply:
Yeah forgot it, my bad.